Avoid conflicts on remerge.
Only provide mesa-classic drivers here for DragonFly.

--- Makefile.orig	2020-08-13 10:38:14 UTC
+++ Makefile
@@ -89,6 +89,15 @@ MESA_PLATFORMS+=	surfaceless
 MESA_PLATFORMS+=	wayland
 .endif
 
+# gallium drivers are installed separately in graphics/mesa-gallium port
+.if ${OPSYS} == DragonFly
+OPTIONS_EXCLUDE:=	VAAPI VDPAU
+DRI_DRIVERS:=		R100 R200 I915 I965 SWRAST
+GALLIUM_DRIVERS:=
+VULKAN_DRIVERS:=	# requires render nodes on INTEL # AMD vulkan driver requires llvm
+MESON_ARGS+=		-Dgallium-xvmc=false -Dgallium-xa=false -Dgallium-va=false -Dgallium-vdpau=false
+.endif
+
 MESON_ARGS+=	-Ddri-drivers="${DRI_DRIVERS:ts,:tl}" \
 			-Dgallium-drivers="${GALLIUM_DRIVERS:ts,:tl}" \
 			-Dvulkan-drivers="${VULKAN_DRIVERS:ts,:tl}" \
@@ -97,9 +106,9 @@ MESON_ARGS+=	-Ddri-drivers="${DRI_DRIVER
 # Disable some options
 MESON_ARGS+=	-Dgallium-xa=false \
 		-Dgles1=false \
-		-Dgles2=false \
+		-Dgles2=true \
 		-Dglx=disabled \
-		-Degl=false \
+		-Degl=true \
 		-Dosmesa=none \
 		-Dtools=""
 
